pkgdb/src/flox-flake.cc

```diff
@@ -94,12 +94,24 @@ lockFlake( nix::EvalState &              state,
            const nix::FlakeRef &         flakeRef,
            const nix::flake::LockFlags & lockFlags )
 {
+  /* We want to make sure we don't make nested calls to forks */
+  // static std::atomic_flag inChildSentinel = ATOMIC_FLAG_INIT;
   auto nixLockFlake
     = [&]() { return nix::flake::lockFlake( state, flakeRef, lockFlags ); };
```

Comment:
seems like now we dont need `flox::lockFlake` in the first place?

Reply:
I think that's the gist of it!  I think this is working.